blocks/bricks/sequence_generators.py

```diff
@@ -270,8 +272,18 @@ def cost_matrix(self, application_call, outputs, mask=None, **kwargs):
         feedback = tensor.set_subtensor(
             feedback[0],
             self.readout.feedback(self.readout.initial_outputs(batch_size)))
+
+        # Run the language model
+        if self.language_model:
+            lm_arguments = self.language_model.evaluate(
```

Comment:
Nit-picking, those are not arguments. You can call them results or deliveries, or anything else you want.